README.md

```diff
@@ -28,7 +28,7 @@ If you're on a mac, check out [these notes](#applebeersnake-mac-brew-installed-p
 
 
 ## Cross compiling!
-Compiling non-pure-Python modules or fetching their manylinux wheels is
+Compiling non-pure-Python modules or fetching their many linux wheels is
```

Comment:
`manylinux` is intentionally one word. See: https://github.com/pypa/manylinux

Reply:
My apologies. I hadn't even realized that was a thing. Good to know!